.stylua.toml

```diff
@@ -0,0 +1,6 @@
+column_width = 120
```

Comment:
I'm being a little pendandtic here, but I'd love for you to learn this skill.

Let's remove this commit to just change the filename and instead edit the original commit
`git rebase --interactive HEAD~3`
- Find the latest commit that just changes the name,
- change from `pick` to `drop` to remove this 3rd commit.
- change the commit which creates the `stylua.toml` file, from `pick` to `edit`

Follow the prompts, but this will drop you into editing this commit, 
you can rename the file `mv stylua.toml .stylua.toml`, then git add like normal, follow the git prompts.

At the end you should have a cleaner commit log check with `git log`

BUT you will not be able to push to your branch because you have edited your tree, 
so you will need to `git push origin --force` force a push.

Then comment back in this PR to let me know

Reply:
I think I am going to die xD. I think this enough for now. I did what u said. But... are u using github-cli

Reply:
No, just the normal `git` on the command line

What is your workflow for pulling & pushing changes?

Reply:
I just do `git pull ` and `git push ` in my command line